tests/test_spark.py

```diff
@@ -286,5 +290,51 @@ def test_cli_multiple_spaces():
                           " --class  ", "org.apache.spark.examples.SparkPi"])
 
 
+# Skip DC/OS < 1.10, because it doesn't have support for file-based secrets.
+@pytest.mark.skipif('shakedown.dcos_version_less_than("1.10")')
+@pytest.mark.sanity
+def test_driver_executor_tls():
```

Comment:
Could we make the job rely on a file being served by the driver (i.e. the channel effected by TLS)? This may be more work than necessary because we'd need to enable/check the `--files` flag.